store/modules/flask/index.js

```diff
@@ -11,6 +11,7 @@ const default_state = {
   status_uuid: STATUS.MESSAGE.SERVER_STILL_INITIALIZING,
   simulation_mode: false,
   barcode_manual_mode: false,
+  ignore_next_system_status_if_matching_this_status: null,
```

Comment:
Essentially, the variable `ignore_next_system_status_if_matching_this_status` variable is a workaround store value which is been used as a workaround, to retain the state.

This seems to be something a response obtained via hardware, which sets backend uuid code to 'caliberated', we might need to fix the root cause of the problem and the work-around has to be removed with in a fixed time frame.